cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -82,12 +98,15 @@ func (i *deployCommand) RegisterHook() {
 		Short:   "Deploys a new Kogito Runtime Service into the given Project",
 		Aliases: []string{"deploy"},
 		Long: `deploy-service will create a new Kogito Runtime Service in the Project context. 
-        If the source is provided, the build will take place on the cluster, otherwise you must upload the application binaries via "oc start-build [NAME-binary] --from-dir=target". 
-		Project context is the namespace (Kubernetes) or project (OpenShift) where the Service will be deployed.
-		To know what's your context, use "kogito project". To set a new Project in the context use "kogito use-project NAME".
+	If the [SOURCE] is provided, the build will take place on the cluster.
```

Comment:
this will generate it:

```
deploy-service will create a new Kogito Runtime Service in the Project context. 
	If the [SOURCE] is provided, the build will take place on the cluster.
	If not, you can also provide a dmn/drl/bpmn/bpmn2 file or a directory containing one or more of those files, using the --from-file
	Or you can also later upload directly the application binaries via "oc start-build [NAME-binary] --from-dir=target
			
	Project context is the namespace (Kubernetes) or project (OpenShift) where the Service will be deployed.
	To know what's your context, use "kogito project". To set a new Project in the context use "kogito use-project NAME".

	Please note that this command requires the Kogito Operator installed in the cluster.
	For more information about the Kogito Operator installation please refer to https://github.com/kiegroup/kogito-cloud-operator#kogito-operator-installation.

```